lib/libutils/isoc/isalnum.c

```diff
@@ -4,7 +4,7 @@
  */
 #include <ctype.h>
 
-int __builtin_isalnum(int c)
+int isalnum(int c)
```

Comment:
Comment in the commit "libutils: strip __builtin_ prefix from implementations":
How does this play with the define `isalnum()` in `<ctype.h>`?

Reply:
Well, due to the define, the commit "libutils: strip _builtin prefix from implementations" is effectively a no-op... just a preparation for the following commit actually ("libutils: ctype.h: do not define functions as builtins"). I guess I'd better squash them.

To answer your question below:

> By dropping this flag [-ffreestanding], did you notice any difference in the generated code?

No. The code is totally identical. I compared `tee.dmp` with and without the two libutils patches applied. The only differences I see in tee.dmp are a few symbols are named differently (`__func__.6169` becomes `__func__.6195` etc.).

It is possible however to see one effect of `-ffreestanding` on some made-up example:

```
$ cat test.c
#include <stdio.h>
int main(int argc, char *argv[])
{
        printf("A");
        return 0;
}
$ arm-linux-gnueabihf-gcc -c test.c -o nofreestanding.o
$ arm-linux-gnueabihf-gcc -c test.c -ffreestanding -o freestanding.o
$ arm-linux-gnueabihf-nm freestanding.o | grep -E "(printf|putchar)"
         U printf
$ arm-linux-gnueabihf-nm nofreestanding.o | grep -E "(printf|putchar)"
         U putchar
```
The flag prevented the printf -> putchar optimization. So, better avoid it if we can.